packages/react-components-migration/react-button-shim-v8-v9/Spec.md

```diff
@@ -0,0 +1,63 @@
+# @fluentui/react-button-shim-v8-v9 Spec
```

Comment:
We can probably remove all `Spec.md` files from this PR given that these are shims and not new components, right?

Reply:
As I bring in the shim code, I'll clean up anything that doesn't need to be there.